libretro.cpp

```diff
@@ -112,7 +112,8 @@ void retro_init(void)
    if (environ_cb(RETRO_ENVIRONMENT_GET_SAVE_DIRECTORY, &dir) && dir)
    {
 	  // If save directory is defined use it, otherwise use system directory
-      retro_save_directory = *dir ? dir : retro_base_directory;
+      // retro_save_directory = *dir ? dir : retro_base_directory;
+	  retro_save_directory = dir;
```

Comment:
The comments above should be updated, since they no longer apply with this change.